leaderboardsgg/leaderboard-site
# LBGG[Nuxt3]: SiteNavbar

## What

Ported all the components that make up the `<SiteNavbar />` from `Vue2` to `Vue3`

## Acceptance

### Steps for testing
- Pull the code
- `pnpm install && pnpm dev` to get things running
- Check the modals respond and the inputs work

### Images
#### SiteNavbar
| • | Mobile | Tablet | Desktop |
| --- | --- | --- | --- |
| Before | ![image](https://user-images.githubusercontent.com/17736056/172758334-6d817a98-37fb-4efe-a61a-4401c5008df5.png) | ![image](https://user-images.githubusercontent.com/17736056/172758247-962915f1-df35-4267-9511-423a1168746c.png) | ![image](https://user-images.githubusercontent.com/17736056/172758111-35b43024-bc81-4e67-991d-769a6acb7140.png) |
| After | ![image](https://user-images.githubusercontent.com/17736056/172758518-6027b517-e769-48eb-9e76-fa9379336f67.png) | ![image](https://user-images.githubusercontent.com/17736056/172758568-d79f90bd-c9db-4f54-9779-7e9d8bcf75b8.png) | ![image](https://user-images.githubusercontent.com/17736056/172758623-7c3af145-37e8-40fb-9df4-f57defd817dd.png) |

#### Modals
| • | Mobile | Tablet | Desktop |
| --- | --- | --- | --- |
| Log In Modal | ![image](https://user-images.githubusercontent.com/17736056/172759007-2e54236f-aafb-425d-a567-f6b011cbfa3e.png) | ![image](https://user-images.githubusercontent.com/17736056/172758966-aad1712f-b26a-4fc3-9594-68724bf448ff.png) | ![image](https://user-images.githubusercontent.com/17736056/172758761-d184ec76-2942-40a9-848a-3e229e4c522a.png) |
| Sign Up Modal | ![image](https://user-images.githubusercontent.com/17736056/172759059-9a98d687-25d5-4b01-bb3c-5bfd7221f88c.png) | ![image](https://user-images.githubusercontent.com/17736056/172759082-c002c3dd-4273-4ed9-bc69-42c30b01fc49.png) | ![image](https://user-images.githubusercontent.com/17736056/172759137-d0f4f95a-24f1-4d42-90c7-759d621bfaee.png) |
